src/lib.rs

```diff
@@ -15,11 +15,9 @@
 The Rust client library for [Prometheus](https://prometheus.io/).
 */
 
-#![cfg_attr(feature = "dev", feature(plugin))]
-#![cfg_attr(feature = "dev", plugin(clippy))]
-#![cfg_attr(not(feature = "dev"), allow(unknown_lints))]
-#![cfg_attr(feature = "dev", allow(needless_pass_by_value))]
-#![cfg_attr(feature = "dev", allow(new_without_default_derive))]
+#![allow(unknown_lints)]
```

Comment:
Can you combine these together?

Reply:
Previously we allow unknown_lints when feature is not dev. I guess for now this allow should be removed?

Reply:
done, by using 
```
#![cfg_attr(feature = "cargo-clippy", allow(needless_pass_by_value, new_without_default_derive))]
```